docs/features/plugins/introduction.md

```diff
@@ -0,0 +1,54 @@
+# Plugins Endpoint
+
+![image](https://github.com/danny-avila/chatgpt-clone/assets/110412045/0ba6c38c-4014-4c3a-a671-d2d64bb2490a)
+
+The plugins endpoint opens the door to prompting LLMs in new ways other than traditional input/output prompting.
+
+The first step is using chain-of-thought prompting for using plugins/tools in a fashion mimicing the official ChatGPT Plugins feature.
+
+More than this, you can use this endpoint for changing your conversation settings mid-conversation. Unlike the official ChatGPT site and all other endpoints, you can switch models, presets, and settings mid-convo, even when you have no plugins selected. This is useful if you first want a creative response from GPT-4, and then a deterministic, lower cost response from GPT-3. Soon, you will be able to use PaLM2 and HuggingFace models, all in this endpoint in the same modular manner.
+
+### Roadmap:
+- More plugins and advanced plugin usage
+- More LLMs to choose from for both Thinking and Completion Phases
+- Alternative prompting methods such as Tree-of-Thought
+
+## Using Plugins 
+
+The LLM process when using Plugins is illustrated below.
+
+![Untitled-2023-06-04-1543(1)](https://github.com/danny-avila/chatgpt-clone/assets/110412045/88483ba0-6b09-43dd-9b70-fb9778871c91)
+
+**When you open the settings with the Plugins endpoint selected, you will view the default settings for the Completion Phase.**
+
+Clicking on **"Show Agent Settings"** will allow you to modify parameters for the thinking phase
+
+![image](https://github.com/danny-avila/chatgpt-clone/assets/110412045/3d0a1e23-b111-459a-a1bc-19ff9262c2b6)
+
+---
+
+![image](https://github.com/danny-avila/chatgpt-clone/assets/110412045/87e5ba8a-194b-4ca3-bec4-820f7434c8c8)
+
+- You can specify which plugins you would like to select from by installing/uninstalling them in the Plugin store
+- See this guide on how to create your own plugins (WIP)
+- ChatGPT Plugins, both community-made and official versions, will be available for use in a future update
+
+### Notes
+- Every additional plugin selected will increase your token usage as there are detailed instructions the LLM needs for each one
+- For best use, be selective with plugins per message and narrow your requests as much as possible
+- If you need help coming up with a good plugin prompt, ask the LLM for suggestions before using one!
+- Chain-of-thought prompting (plugin use) will always be more expensive than regular input/output prompting, so be sure
```

Comment:
very very minor but was this sentence cut off?

Reply:
good catch!